accounting/receivables/customer_invoices/payment_terms.rst

```diff
@@ -1,96 +1,109 @@
+=============
 
-- **an invoice**: to apply the payment term on this invoice only
+Go to :menuselection:`Accounting --> Configuration --> Payment Terms` and click on *Create*.
 
-If an invoice has a payment term, the journal entry related to the
-invoice is different. Without payment term or tax, an invoice of $100
-will produce this journal entry:
+The **Description on the Invoice** is the text that is displayed on the sale order, invoice, or bill.
 
-+----------------------+------------+---------+----------+
-| Account              | Due date   | Debit   | Credit   |
-+======================+============+=========+==========+
-| Account Receivable   |            | 100     |          |
-+----------------------+------------+---------+----------+
-| Income               |            |         | 100      |
-+----------------------+------------+---------+----------+
+In the **Terms** section, you can add a set of rules, or terms, that define what needs to be paid,
+and by which due date.
 
-If you do an invoice the 1st of January with a payment term of 10%
-within 3 days and the balance within 30 days, you get the following
-journal entry:
+To add a term, click on *Add a line*, and define its *Type*, *Value*, and *Due Date Computation*.
 
-+----------------------+------------+---------+----------+
-| Account              | Due date   | Debit   | Credit   |
-+======================+============+=========+==========+
-| Account Receivable   | Jan 03     | 10      |          |
-+----------------------+------------+---------+----------+
-| Account Receivable   | Jan 30     | 90      |          |
-+----------------------+------------+---------+----------+
-| Income               |            |         | 100      |
-+----------------------+------------+---------+----------+
+.. important::
+   - Terms are computed in the order they are set up.
+   - The **balance** should always be used for the last line.
 
-In the customer statement, you will see two lines with different due
-dates.
+In the following example, 30% of the invoice is due on the day of issuance of the invoice, and the
+balance is due at the end of the following month.
 
-Payment terms for vendor bills
-------------------------------
+.. image:: media/payment_terms_configuration.png
+   :align: center
+   :alt: Example of payment terms. The last line is the balance due the 31st of the following month.
 
-The easiest way to manage payment terms for vendor bills is to record a
-due date on the bill. You don't need to assign a payment term, just the
-due date is enough.
+Using Payment Terms
+===================
 
-But if you need to manage vendor terms with several installments, you
-can still use payment terms, exactly like in customer invoices. If you
-set a payment term on the vendor bill, you don't need to set a due date.
-The exact due date for all installments will be automatically created.
+Payment Terms can be defined with the **Payment Terms** field on:
+
+- | **Contacts**
```

Comment:
```suggestion
- | **Contacts**: to set specific...
```

for all topics (continue in the same line?)

Reply:
Agreed, I changed them to continue on the same line

Reply:
Although, I'd rather be consistent with the formatting used at the beginning of this doc...
So I changed the first list in the intro and put the text in bold and I keep the separate lines.
I think it looks better like this.

I also tried to use [definition lists](https://docutils.sourceforge.io/docs/user/rst/quickref.html#definition-lists), but it doesn't work that well either.